wgpu-hal/src/gles/queue.rs

```diff
@@ -1330,6 +1330,7 @@ impl super::Queue {
                         unsafe { gl.disable(glow::DEPTH_CLAMP) };
                     }
                 }
+                unsafe { gl.polygon_mode(glow::FRONT_AND_BACK, state.polygon_mode) };
```

Comment:
I think these should be:
```rust
if (!gl.version().is_embedded ) {
    unsafe { gl.polygon_mode(glow::FRONT_AND_BACK, state.polygon_mode) };
}
```

Reply:
Yea didn't thought of it, as for WebGL it was just ignored, but calling it on GLES logically results in an invalid op